components/Project/index.js

```diff
@@ -25,6 +41,9 @@ export default function Project({ project, favorites, onToggleFavorite }) {
         title="Instructions"
         text={project.instructions}
       ></ProjectInfoBox>
+      <Link href={`${project._id}/edit`}>
+        <StyledButton>Edit</StyledButton>
+      </Link>
```

Comment:
please don't put a button inside a Link. You can style the Link itself by using `const `StyledLink = styled(Link)` and then append the styling in the template string.

Reply:
Thank you for your review :) We applied the suggestions and put the Link inside a div called "ButtonSection" because there will be another button for delete.